content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,211 @@ title: Instrumentation
 weight: 3
 ---
 
-This guide will cover creating and annotating spans, creating and annotating metrics, how to pass context, and a guide to automatic instrumentation for JavaScript. This simple example works in the browser as well as with Node.js
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+If a `TracerProvider` is not created, the OpenTelemetry APIs for tracing will
+use a no-op implementation and fail to generate data.
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+### Node.js
+
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-trace-node
+```
+
+Next, create a separate `tracing.js` file that has all the SDK initialization
+code in it:
+
+```js
+const opentelemetry = require("@opentelemetry/api");
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { ConsoleSpanExporter, BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
+
+const provider = new NodeTracerProvider();
+const exporter = new ConsoleSpanExporter();
+const processor = new BatchSpanProcessor(exporter);
+provider.addSpanProcessor(processor);
+
+// Optionally register automatic instrumentation libraries
+registerInstrumentations({
+  instrumentations: [],
+});
+
+provider.register();
+
+module.exports.tracer = opentelemetry.trace.getTracer(
+  'name-of-tracer-goes-here'
+);
+```
+
+Next, ensure that `tracing.js` is required in your node invocation. For example:
 
-function doWork() {
-  console.log('work...');
-  // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
-}
+```
+node --require './tracing.js' <app-file.js>
 ```
 
-## Initializing a Tracer
+### Browser
```

Comment:
This looks similar to https://opentelemetry.io/docs/instrumentation/js/getting-started/browser/, is the goal to replace that one and have browser/node.js in a single file?

Reply:
The goal is to show both of the major ways to initialize tracing. Since each uses a different SDK, I think it's better to have both sections here.

Reply:
OK. Should we then get rid of the browser getting started?

Reply:
I don't think so, no. The goal here is not to be DRY. I think it's fine to have some duplicated code in different contexts.